client/utils/events.ts

```diff
@@ -419,9 +419,9 @@ const getDateStringForEvent = (event, dateOnly = false, useLocal = true, withTim
     // !! Note - expects event dates as instance of moment() !! //
     const dateFormat = appConfig.planning.dateformat;
     const timeFormat = appConfig.planning.timeformat;
-    const start = get(event.dates, 'start');
-    const end = get(event.dates, 'end');
-    const tz = get(event.dates, 'tz');
+    const start = get(event, 'start');
```

Comment:
This function is used in multiple places, passing in the entire Event item.
In this case it might be good to change those usages of `getDateStringForEvent` as well.